charts/drone-vault-secrets/templates/rbac.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "drone-vault-secrets.fullname" . }}
+  labels:
+    {{- include "drone-vault-secrets.labels" . | nindent 4 }}
+
+{{/* A Role is created for the namespace specified in rbac.secretNamespace. */}}
```

Comment:
Why does the vault-secrets extension need access to kubernetes secrets? At most it needs auth credentials to communicate with vault, but why would it need access to other secrets in the namespace?

Reply:
Looks like the service-account needs a `ClusterRoleBinding` to the `system:auth-delegator` role: https://www.vaultproject.io/docs/auth/kubernetes#configuring-kubernetes